src/OpenTelemetry.Instrumentation.StackExchangeRedis/Implementation/RedisMetrics.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics.Metrics;
+using System.Reflection;
+using OpenTelemetry.Internal;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Instrumentation.StackExchangeRedis.Implementation;
+
+internal class RedisMetrics : IDisposable
+{
+    internal const string DurationMetricName = "db.client.operation.duration";
+    internal const string QueueTimeMetricName = "db.client.operation.queue_time";
+    internal const string ServerTimeMetricName = "db.client.operation.server_time";
+
+    internal static readonly Assembly Assembly = typeof(StackExchangeRedisInstrumentation).Assembly;
+    internal static readonly AssemblyName AssemblyName = Assembly.GetName();
+    internal static readonly string InstrumentationName = AssemblyName.Name!;
+    internal static readonly string InstrumentationVersion = Assembly.GetPackageVersion();
```

Comment:
Any reason why is this diff. than https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInMetricsListener.cs#L27 ? Not sure which one is apt here, but its best to stick with existing practices.

Reply:
I opted for the one that I saw being used most recently. I'm also not sure which one should be used and why there are variations.

Reply:
`Assemlby.GetPackageVersion()` is correct. `AssemblyName.Version!.ToString();` always return 1.0.0.0.
As I remember, we have decided to keep it as is, due to migration to natively implemented metrics by .NET team.

Ref: #1624